src/scrapers/base-isracard-amex.ts

```diff
@@ -392,19 +346,19 @@ class IsracardAmexBaseScraper extends BaseScraperWithBrowser<ScraperSpecificCred
 
   async login(credentials: ScraperSpecificCredentials): Promise<ScraperScrapingResult> {
     await this.page.setRequestInterception(true);
-    this.page.on('request', request => {
+    this.page.on('request', (request) => {
       if (request.url().includes('detector-dom.min.js')) {
         debug('force abort for request do download detector-dom.min.js resource');
-        void request.abort(undefined, interceptionPriorities.abort);
+        void request.abort();
       } else {
-        void request.continue(undefined, interceptionPriorities.continue);
+        void request.continue();
       }
     });
 
-    await maskHeadlessUserAgent(this.page);
-
+    debug('navigate to login page');
     await this.navigateTo(`${this.baseUrl}/personalarea/Login`);
```

Comment:
```suggestion
    await this.navigateTo(`${this.baseUrl}/personalarea/Login`, 'networkidle0');
```

Can you see if this change works instead of calling `sleep`? (`'networkidle2'` might also work, but `'networkidle0'` will wait more time)

Reply:
my fork wasn't synced, retried after updating it, it doesn't solves the issue. i tiried both "networkidle2" and "networkidle0"